### ISSUE TITLE
About the environment.yaml

### ISSUE THREAD

**Issue by GreameLee** [wustl-cig/DOLCE#1]
Could we have a requirement.txt to pip install? Using conda to install leap is very hard

**Comment by JiamingLiu-Jeremy** [comment]
Hi, thank you for interesting in our work. Have you tried install using "install_envs.sh". Once have a pytorch environment in conda, the leap is installed from source.

**Comment by GreameLee** [comment]
yes, I have tried it but the dependency has conflicts and it can not create the dolce in conda

**Comment by JiamingLiu-Jeremy** [comment]
Hi, can you please share your hardware/linux environment settings etc ? The current version tested well on NVIDIA V100, 3090 and A6000, with gcc 8.5 and CUDA 11.1.

**Comment by dewan85** [comment]
Hello, is it possible to use this code in windows 10?

**Comment by GreameLee** [comment]
> Hello, is it possible to use this code in windows 10?

I have tried but failed

**Comment by JiamingLiu-Jeremy** [comment]
Hi, the MPI might be the issue for environments inconsistency. Have you tried to run on singe GPU without using DDP at inference ?

**Comment by wdayang** [comment]
I also tried to create the environment using install_envs.sh, but it shows conflicts then fails. Actually, there are many packages to be installed, I believe only a few is used. By any chance, Is it possible to pick out the most key packages, so that the installation might be more friendly @JiamingLiu-Jeremy ? Thank you.
